packages/editor/src/lib/editor/managers/SnapManager.ts

```diff
@@ -12,12 +11,12 @@ import {
 	Vec2d,
 	VecLike,
 } from '@tldraw/primitives'
-import { TLLineShape, TLParentId, TLShape, TLShapeId, Vec2dModel } from '@tldraw/tlschema'
-import { compact, dedupe, deepCopy } from '@tldraw/utils'
+import { TLParentId, TLShape, TLShapeId, Vec2dModel } from '@tldraw/tlschema'
+import { dedupe, deepCopy } from '@tldraw/utils'
 import { atom, computed, EMPTY_ARRAY } from 'signia'
 import { uniqueId } from '../../utils/data'
 import type { Editor } from '../Editor'
-import { getSplineForLineShape, LineShapeUtil } from '../shapes/line/LineShapeUtil'
```

Comment:
We want to get rid of these types of hard-coded dependencies on specific shapes. _**tldraw zero**_